catberry_components/head/head.dust

```diff
@@ -1,3 +1,4 @@
 <meta charset="UTF-8">
 <title>{.title}</title>
-<script src="bundle.js"></script>
+<script src="app.js"></script>
+<script src="externals.js"></script>
```

Comment:
Does it work this way? Because the order should be externals, app not the other way around.